<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1306</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="menuLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>621</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QListWidget" name="mainListView">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>721</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>130</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="menuButton">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>60</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>MENU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ON/OFF</string>
    </property>
   </widget>
   <widget class="Battery" name="battery" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>120</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="breathPacersetting" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>621</width>
      <height>401</height>
     </rect>
    </property>
    <widget class="QSlider" name="breathSlider">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>451</width>
       <height>51</height>
      </rect>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="breathSliderText">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Breath Pacer: 10 seconds</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sessionScreen" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>711</width>
      <height>471</height>
     </rect>
    </property>
    <widget class="LightStrip" name="breathPacerLights" native="true">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>410</y>
       <width>561</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCustomPlot" name="graphWidget" native="true">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>511</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="coherenceScoreLabel">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>120</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Coherence Score: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lengthLabel">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>90</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Length:</string>
     </property>
    </widget>
    <widget class="QLabel" name="achievementScoreLabel">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Achievement Score:</string>
     </property>
    </widget>
    <widget class="QLabel" name="percentHighLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Percentage of Time in High Coherence: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="percentMediumLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Percentage of Time in Medium Coherence: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="percentLowLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Percentage of Time in Low Coherence: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="averageCoherenceScoreLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Average Coherence Score:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="dateLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Date Created: </string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteSessionButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>440</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Session</string>
     </property>
    </widget>
    <widget class="LED" name="coherenceLight" native="true">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="selectorButtons" native="true">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>260</y>
      <width>341</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QPushButton" name="upButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>UP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="leftButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>LEFT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rightButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>RIGHT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="downButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>DOWN</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="selectButton">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>350</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>SELECT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hrvSignalButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>HRV Signal</string>
    </property>
   </widget>
   <widget class="LED" name="signalLight" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <zorder>selectorButtons</zorder>
   <zorder>breathPacersetting</zorder>
   <zorder>menuLabel</zorder>
   <zorder>mainListView</zorder>
   <zorder>backButton</zorder>
   <zorder>menuButton</zorder>
   <zorder>powerButton</zorder>
   <zorder>battery</zorder>
   <zorder>sessionScreen</zorder>
   <zorder>selectButton</zorder>
   <zorder>hrvSignalButton</zorder>
   <zorder>signalLight</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1306</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Battery</class>
   <extends>QWidget</extends>
   <header>battery.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LightStrip</class>
   <extends>QWidget</extends>
   <header>lightstrip.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LED</class>
   <extends>QWidget</extends>
   <header>led.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
